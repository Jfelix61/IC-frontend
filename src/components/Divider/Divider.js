import React from "react";
import "./Divider.scss";
import PropTypes from "prop-types";

const Divider = ({ className }) => {
  return <div className={`divider ${className ? className : ""}`}>&#8203;</div>;
};

Divider.propTypes = {
  className: PropTypes.string
};

export default Divider;
