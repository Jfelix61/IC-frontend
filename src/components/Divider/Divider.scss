.divider {
  width: 2px;
  background-color: var(--gray);
}
