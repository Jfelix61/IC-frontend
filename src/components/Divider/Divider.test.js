import React from "react";
import { render, getByTestId } from "react-testing-library";

import Divider from "./Divider";

describe("Divider", () => {
  it("renders without crashing", () => {
    const { asFragment } = render(<Divider />);
    expect(asFragment()).toMatchSnapshot();
  });
});
