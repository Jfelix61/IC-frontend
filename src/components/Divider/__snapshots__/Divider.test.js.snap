// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider renders without crashing 1`] = `
<DocumentFragment>
  <div
    class="divider "
  >
    â€‹
  </div>
</DocumentFragment>
`;
